src/pages/search/index.tsx

```diff
@@ -465,6 +484,50 @@ const Search: InferGetServerSidePropsType<typeof getServerSideProps> = ({
                       )}
                     </SlideOut>
                   )}
+                  <SlideOut showCloseButton={false}>
```

Comment:
Suggestion: see how with the `ConceptPicker` above on line `481` could we potentially move the contents of the slideout into a separate component? And also while doing that move the associated logic? Would be nice to make the search page smaller

Reply:
Couple of reasons I see for this:
1. Any state logic can be encapsulated within the separate component rather than adding more to the search page (already a far too big component imo @ > 700 lines). And more broadly the behaviour can be managed separately to the search page
2. Managing changes to the component becomes more traceable and less chance for conflicts
3. Ideally the search page would only consist of core search logic and all state and interactivity concerns would be managed in the appropriate components, we aren't there now but that's the ambition. Moving this into a separate component I think gets us a little closer to that.